src/PKSim.CLI.Core/PKSim.CLI.Core.csproj

```diff
@@ -29,6 +29,8 @@
   <ItemGroup>
     <PackageReference Include="OSPSuite.Utility" Version="4.0.0.4" />
     <PackageReference Include="OSPSuite.Core" Version="9.0.128" />
+    <PackageReference Include="Serilog.Extensions.Logging" Version="3.0.1" />
+    <PackageReference Include="Serilog.Extensions.Logging.File" Version="2.0.0" />
```

Comment:
@abdelr Wondering why the core requires a reference. Isn't is something the caller is referencing and adding to the factory?

Reply:
@msevestre @georgeDaskalakis Indeed it is not required. Maybe we add this at some experimental step. I just removed it and will be updated in the next commit.